bin/sandbox

```diff
@@ -38,7 +38,7 @@ if [[ -z $GENERATE_FRONTEND ]]; then
   extension_gem_line="gem '$extension_name', path: '..'"
 else
   sandbox_name='sandbox-generated'
-  extension_gem_line=""
+  extension_gem_line="gem '$extension_name', path: '..', require: false"
```

Comment:
This allows us to run `solidus_starter_frontend` from the sandbox directory. However, we set `require: false` so that the gem's engine is not loaded in the sandbox.